trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Drag/parasite_total.py

```diff
@@ -61,6 +61,13 @@ def parasite_total(state,settings,geometry):
         conditions.aerodynamics.drag_breakdown.parasite[propulsor.tag].parasite_drag_coefficient  = parasite_drag * ref_area/vehicle_reference_area * propulsor.number_of_engines
         total_parasite_drag += parasite_drag * ref_area/vehicle_reference_area * propulsor.number_of_engines
 
+    # from pylons
```

Comment:
If the pylon script is valid for supersonics it should be used as a default. You need to modify your concorde setup script to skip this if you don't want to use it.